FAQs.md

```diff
@@ -0,0 +1,165 @@
+# APlayer FAQs
+
+
+This is the list of Frequently asked questions for APlayer. It is intended to serve as a self-service utility document. This is to aid new users or already existing users of APlayer. This document may be updated from time to time to suit any new changes to the software as new developments or changes will be implemented.
+
+
+1. **What is APlayer?**
+
+APlayer is a  basic music player application for playing media(music) files on your mobile device.
+
+---
+
+2. **What are the features of APlayer?**
+
+APlayer is designed with all the basic features of a music player. Aside from that, it has some other cool features like; 
+
+- Playlist.
+- Equalizer.
+- Grouped menus. 
+- File browser.
+- Lyrics
+- Gesture 
+- UI customizations.
+
+---
+
+3. **How do I create, and add songs to a playlist?**
+
+To create a playlist on APlayer;
+
+- navigate to the playlists page. To do that; swipe left or right depending on the menu you are on. 
+- On the playlists page, click on the plus (+) sign to create a new playlist. 
+- Enter a suitable name for the playlist and click create. 
+- After the playlist has been created, you'll be prompted to add songs to it immediately. You must add at least 1 song for the playlist to be created successfully. 
+- If you wish to add songs to a playlist at other times, highlight a song by pressing and holding it.
+- Mark the songs you wish to add to the playlist.
+- At the top part of the app, you will find 3 icons. Click on the second icon(icon in the middle) to add the songs to a playlist.
+-  Select the playlist, and the song will be added to it. 
+
+---
+
+4. **Can I rename a playlist?**
+
+Yes, you can rename a playlist at any time. 
+
+- Access the playlist page.
+- Click the 3 dots on the playlist you wish to change its name.
+- Click on the rename option.
+- Input the new name and confirm the change. 
+
+---
+
+5. **Can I delete unwanted playlists?**
+
+Yes, you can delete an unwanted playlist. To delete a playlist;
+
+- Perform the same action as though you want to rename a playlist
+- Instead of clicking the rename option, click on remove. 
+
+---
+
+6. **Can I import and export playlists on Aplayer?**
+
+Yes, you can easily import and export playlists on APlayer. This option is found in the settings page. 
+
+---
+
+7. **Can I sort tracks on APlayer?**
+
+No, you can't sort tracks on the app at the moment. But, this feature will be implemented soon. 
+
+---
+
+8. **Where can I find the APlayer's equalizer?**
+
+APlayer doesn't have its own equalizer. It uses your device's equalizer app. But you can access it from/on APlayer. The equalizer is located on the settings page. 
```

Comment:
built-in equalizer will be added on next release

Reply:
So for this, I just have to write that the app has an equalizer? Because I don't know when the next release will be available. Or maybe when the new release is out, I can edit the document then.